### DISCUSSION TITLE
Channel 'order' differences between fonts

### DISCUSSION THREAD

**Discussion by LrsChrSch** [Chlumsky/msdf-atlas-gen#111] [Q&A]
Hey there!

Found this awesome project today and it definitely piqued my interest. Haven't had the time to go in depth into all the research papers yet so this question might be obsolete lol. Figured I'd ask anyway, maybe someone quickly has the answer and can help me out :)

![font1](https://github.com/user-attachments/assets/b625a0a2-48e2-4825-9795-00e9a962df02)
Font 1 (Regular)

![inter](https://github.com/user-attachments/assets/2ca24986-666d-4721-a70a-4b5d0425d782)
Font 2 (Regular) - Inter

![inter-bold](https://github.com/user-attachments/assets/894d3105-b3fb-4184-8a67-025893977e5f)
Font 2 (Bold) - Inter

Both of these fonts share some common characteristics. Not identical ofc but at least similar

Now notice the pattern to the sides of the parantheses:
blue-green-blue-green for the first one
blue-green-green-blue for the second one (on both weights)

The 'V' uses blue-red on the outside of the first one and green-blue for the other two

I can't figure out why the first one would use the one color channel first and then the other after that while the other font has it the other way around. It doesn't seem to be random since the different weights of Inter look almost the same.

Is there a way to unify the use of these color channels across different fonts?

I know that for any practical purpose it doesn't matter in the slightest. Both fonts look the way they should when rendering them. But I thought about maybe playing around with those color channels to get some weird abstract glitches or something and for that it would be helpful if this 'assignment' would match between different fonts.

I know it's a weird request, so if you have any questions about what I mean, I'll try to clarify.

Thank you so much in advance!

**Comment by Chlumsky** [comment]
Sure, all you have to do is use `-coloringstrategy simple -seed 0` and then the matching glyphs have to have the same set of contours in the same order, each beginning at the same vertex, and having the same set of vertices identified as corners (can be adjusted by `-angle`).
